peps/pep-0792.rst

```diff
@@ -0,0 +1,371 @@
+PEP: 792
+Title: Project status markers in the simple index
+Author: William Woodruff <william@yossarian.net>,
+        Facundo Tuesca <facundo.tuesca@trailofbits.com>
+Sponsor: Donald Stufft <donald@stufft.io>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Discussions-To: https://discuss.python.org/t/79356
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 08-Apr-2025
+Post-History: `03-Feb-2025 <https://discuss.python.org/t/79356/>`__
+
+Abstract
+========
+
+This PEP proposes a standardized set of project status markers, as well
+as a mechanism for communicating those markers in the HTML and JSON
+simple indices.
+
+Rationale and Motivation
+========================
+
+The "status" of a project is an important piece of metadata, made more important
+by growth in both the breadth and depth of the Python packaging ecosystem.
```

Comment:
or 'scope', etc. Not sure which dimensions 'breadth' vs 'depth' would cover!
```suggestion
by growth in both the expanse of the Python packaging ecosystem.
```

Reply:
"Breadth and depth" was meant to be "number of packages" and "depth of dependency trees" respectively, but I can rephrase to make that clearer!

Reply:
Went with "size and complexity," but I'm happy to rephrase as needed.